app/models/nic/interface.rb

```diff
@@ -57,13 +57,13 @@ def alias?
     end
 
     def fqdn_changed?
```

Comment:
Since these two methods are meant to mimic active record methods, should we also change them to the new naming?

Reply:
I thought its use were more spread over the code, but its indeed only in this file, so renaming was no problem 👍.